src/us/kbase/auth2/lib/Authentication.java

```diff
@@ -269,10 +270,13 @@ public synchronized void updateConfig() throws AuthStorageException {
 	 * @param pwd the new password for the root account.
 	 * @throws AuthStorageException if updating the root account fails.
 	 */
-	public void createRoot(final Password pwd) throws AuthStorageException {
+	public void createRoot(final Password pwd) throws AuthStorageException, IllegalPasswordException {
 		nonNull(pwd, "pwd");
+		pwd.checkValidity();
```

Comment:
yeah, but could be a pwd from another service they put in accidentally. Low risk, but might as well do the right thing.